src/components/overview-header/overview-header.js

```diff
@@ -148,6 +180,25 @@ class OverviewHeader extends React.PureComponent {
         </div>
       </div>
     );
+
+    if (card) {
+      return (
+        <div
+          className="flex overflow-hidden round-bold mt18 mb30"
+          style={{ boxShadow: cardBoxShadow }}
+        >
+          {card && (
+            <div
+              className={`flex-child-no-shrink bg-${highlightColor}`}
+              style={{ width: 7 }}
```

Comment:
```suggestion
              className={`w6 flex-child-no-shrink bg-${highlightColor}`}
```

Reply:
Same, even one pixel changes the presentation of this highlighting bar and I think it's acceptable to leave a style prop here.